cherab/openadas/rates/atomic.pyx

```diff
@@ -33,16 +36,15 @@ cdef class IonisationRate(CoreIonisationRate):
         # unpack
         ne = data['ne']
         te = data['te']
-        rate =  np.log10(data['rate'])
+        rate = np.log10(data['rate'])
 
         # store limits of data
         self.density_range = ne.min(), ne.max()
         self.temperature_range = te.min(), te.max()
 
         # interpolate rate
-        self._rate = Interpolate2DCubic(
-            ne, te, rate, extrapolate=extrapolate, extrapolation_type="quadratic"
-        )
+        extrapolation_type = 'nearest' if extrapolate else 'none'
```

Comment:
I'd use linear, not nearest in these classes. The rates should taper to zero. Nearest will prevent this.

Reply:
I agree that extrapolating the rates linearly makes more physical sense. Although linear extrapolation is safe to use for PEC, radiated power and beam rates, the rates of ionisation, recombination and thermal CX are interpolated and extrapolated on a log scale:
https://github.com/cherab/core/blob/ce75f778467b13ecb369b2a2bcce7ad422d4016a/cherab/openadas/rates/atomic.pyx#L36
and using linear extrapolation for exponent may be unsafe. Take for example the Be<sup>+</sup> recombination rate at Te = 1 eV. With linear extrapolation, we'll have **6.88 10<sup>-16</sup>** m<sup>3</sup> s<sup>-1</sup> for Ne = 10<sup>**22**</sup> m<sup>-3</sup> and **2.03 10<sup>6</sup>** m<sup>3</sup> s<sup>-1</sup> for Ne = 10<sup>**23**</sup> m<sup>-3</sup>.

Should these atomic rates be interpolated on a linear scale just like PEC and beam rates to avoid this problem?